src/config/logging.rs

```diff
@@ -11,22 +11,26 @@ const LOG_FILE_PREFIX: &str = "log";
 const LOG_FILE_DIRECTORY: &str = "./log";
 
 pub fn configure_tracing() -> anyhow::Result<Option<WorkerGuard>> {
-    let mut guard: Option<WorkerGuard> = None;
+    let mut _guard: Option<WorkerGuard> = None;
```

Comment:
Underscore prefix to make the linter happy - indicates that we don't necessarily intend for this to be read anywhere